### DISCUSSION TITLE
Does mmDeploy support DAB-DETR?

### DISCUSSION THREAD

**Discussion by Zaughter** [open-mmlab/mmengine#1329] [Q&A] [answered]
As the title, can i use mmDection and mmDeploy to get my DAB-DETR onnx file?

**Comment by RunningLeon** [comment]
@Zaughter hi, mmdet has  [dab-detr](https://github.com/open-mmlab/mmdetection/tree/main/configs/dab_detr) ckpt, but mmdeploy has not supported its `torch2onnx` by far. You can check the [support-model-list](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html#supported-models).

**Comment by RunningLeon** [comment, marked as answer]
@Zaughter hi, could try this PR https://github.com/open-mmlab/mmdeploy/pull/2410 to deploy dab-detr
